trac-env/templates/custom_ticket.html

```diff
@@ -9,8 +9,12 @@ <h2>Please read this first:</h2>
         Contact <a href="mailto:security@djangoproject.com">security@djangoproject.com</a> instead.</strong>
     </li>
     <li>
-      Report bugs and suggest features in Django itself (or its
-      documentation) using this ticket tracker.
+      Report bugs and suggest documentation changes using this ticket tracker.
+    </li>
+    <li>
+      New feature proposals should first be discussed on the <a href="https://forum.djangoproject.com/">Django Forum</a>.
```

Comment:
I think it makes sense to link directly to the "Django Internals" section of the forum:

```suggestion
      New feature proposals should first be discussed on the <a href="https://forum.djangoproject.com/c/internals/5">Django Forum</a>.
```

Reply:
Indeed 👍